PDF_UA/2/8.2 Logical structure/8.2.5 Additional requirements for specific structure types/8.2.5.20 Link and Reference/verapdf-profile-8-2-5-20-t01.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.2 Logical structure - 8.2.5.20 Link and Reference - Link annotation</name>
+        <description>A link annotation and its associated content shall be enclosed in either a Link or Reference
+            structure element</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDLinkAnnot">
+            <id specification="ISO_14289_2" clause="8.2.5.20" testNumber="1"/>
+            <description>A link annotation and its associated content shall be enclosed in either a Link or Reference
+                structure element</description>
+            <test>structParentStandardType == 'Link' || structParentStandardType == 'Reference'</test>
+            <error>
+                <message>A Link annotation is artifact or is nested within %1 tag (standard type = %2) instead of Link 
```

Comment:
A Link annotation is marked as Artifact or is nested within %1 tag (standard type = %2) instead of Link or Reference